sw/device/tests/aes/aes_test.c

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/aes.h"
+#include "sw/device/lib/common.h"
+#include "sw/device/lib/uart.h"
+
+// The example below is extracted from Appendix C of
```

Comment:
Example below, the entire implementation?

Reply:
No, it's just the plaintext, key and the golden ciphertext that are extracted from the FIPS document. I clarified the comment.